### PR TITLE
extra_tests_gnome: schedule wine and chrome tests on x86_64 only

### PR DESCRIPTION
**Pull request by ggardet**
- related ticket: https://progress.opensuse.org/issues/77653

### CODE REVIEW

**Comment by asdil12** [comment]
Maybe it would be easier to read if you would combine steam and wine into one conditional schedule called `x86_64-tests`

**Comment by asdil12** [comment]
You could add chrome to that tests, as it's failing on arm according to https://progress.opensuse.org/issues/77653 as well.

**Review by asdil12 — APPROVED** [review]
LGTM

File: schedule/functional/extra_tests_gnome.yaml [hunk 1]
```diff
@@ -20,20 +20,20 @@ conditional_schedule:
                 - x11/keyboard_layout_gdm
             'x86_64':
                 - x11/keyboard_layout_gdm
-    steam:
+    x86_64_tests:
         ARCH:
             'x86_64':
                 - x11/steam
+                - x11/wine
+                - x11/chrome
```
Comment (asdil12) [inline schedule/functional/extra_tests_gnome.yaml:28]: You seem to have some trailing whitespaces that make the testsuite fail.

Comment (ggardet) [reply schedule/functional/extra_tests_gnome.yaml]: Fixed.

**Pull request merged**